.env.development

```diff
@@ -1 +1,2 @@
 REACT_APP_GA4_ID=G-4S06BFBSX6
+REACT_APP_API_DOMAIN=https://sos-api.ugtf.dev/live/api/v1/sms
```

Comment:
What do you think about moving URI to the code?
REACT_APP_API_DOMAIN=https://sos-api.ugtf.dev/live/api/v1/
and sms as path in the code?

Reply:
would not say that i'm 100% convinced :)
maybe when we have at >= 2 different endpoints there

let me know if you feel strongly about it and then i'll send a separate PR to do it your way